backend/compact-connect/lambdas/nodejs/email-notification-service/lambda.ts

```
@@ -386,6 +386,40 @@ export class Lambda implements LambdaInterface {
+            if (!event.specificEmails?.length) {
+                throw new Error('No recipients found for privilege expiration reminder email');
+            }
+            if (!event.templateVariables?.providerFirstName
+                || !event.templateVariables?.expirationDate
+                || !event.templateVariables?.privileges) {
+                throw new Error('Missing required template variables for privilegeExpirationReminder template.');
+            }
+            const privileges = event.templateVariables.privileges as Array<{ jurisdiction?: string; 
+                licenseType?: string; privilegeId?: 
+                string; dateOfExpiration?: string;
+                formattedExpirationDate?: string }>;
```

Biome (2.3.13)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Safe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)